api/v1/serializers/notes-serializer.js

```diff
@@ -29,6 +30,7 @@ const serializeNotes = (rawNotes, query) => {
     resourcePath: noteResourcePath,
     topLevelSelfLink,
     keyForAttribute,
+    enableDataLinks,
```

Comment:
Besides `topLevelSelfLink`, the `serializerArgs` are exactly the same in `serializeNotes` and `serializeNote` method. Do you think we can do something similar like [here](https://github.com/osu-mist/new-students-api/blob/feature/CO-1340-remake-students-api/api/v1/serializers/students-serializer.js#L19-L34)?

Reply:
I merged in the changes from the skeleton which dealt with the uri-builder changes and refactored notes-serializer